dapp/src/components/shared/ProgressBar.tsx

```diff
@@ -1,16 +1,28 @@
 import React from "react"
-import { Box, Progress, ProgressProps, ProgressLabel } from "@chakra-ui/react"
+import { Progress, ProgressProps, ProgressLabel, Icon } from "@chakra-ui/react"
+import { BoltFilled } from "#/assets/icons"
 
 function ProgressBar(props: ProgressProps) {
-  const { value, children, ...restProps } = props
+  const { value = 0, children, ...restProps } = props
 
   return (
     <Progress value={value} hasStripe {...restProps}>
-      <ProgressLabel>
-        {children}
+      <ProgressLabel>{children}</ProgressLabel>
 
-        <Box>{value}%</Box>
-      </ProgressLabel>
+      {value > 2 && (
```

Comment:
Why greater than `2`? This is a shared component so are we sure it will be ok for other use cases?

Reply:
Ref commit: c51c695c